samcli/commands/build/command.py

```diff
@@ -101,6 +107,23 @@
     help="If your functions depend on packages that have natively compiled dependencies, use this flag "
     "to build your function inside an AWS Lambda-like Docker container",
 )
+@click.option(
+    "--container-env-vars",
+    "-e",
+    default=None,
+    multiple=True,  # Can pass in multiple env vars
+    required=False,
+    help="Input environment variables through command line to pass into build containers, you can either "
+    "input function specific format (FuncName.VarName=Value) or global format (VarName=Value). Example: "
+    "sam build --use-container --container-env-vars Func1.VAR1=value1 --container-env-vars VAR2=value2",
+)
+@click.option(
+    "--container-env-vars-file",
```

Comment:
`sam local invoke` has `--container-env-vars` as a path to a file. It would be more consistent to make this `--container-env-vars` as well, and use the singular form for direct input.